resources/articles/sdks/javascript.md

```diff
@@ -8,212 +8,24 @@
 
 <spid-toc></spid-toc>
 
-## Read more about the JavaScript SDK
+## Read more about the JavaScript SDK for browsers
 
-- [Events](/sdks/js/events/)
-- [Response signature and validation](/sdks/js/response-signature-and-validation/)
-- [API Docs](/sdks/js/api-docs/)
-- [Hosting](/sdks/js/hosting/)
-- [Best practices](/sdks/js/best-practices/)
+- [Check out the Github repo](https://github.schibsted.io/spt-identity/spid-sdk-browser)
+- [API/Getting started docs](https://pages.github.schibsted.io/spt-identity/spid-sdk-browser)
```

Comment:
Unrelated to this PR, but since it's linked from here now: the index page of that documentation only contains "Hello, world!" (there's a better intro on a [subpage](https://pages.github.schibsted.io/spt-identity/spid-sdk-browser/tutorial-01-overview.html)). 🙂

Reply:
Yes, those docs are in... a lot of flux right now. Working full speed towards improving both the docs, and making sure the API surface in the SDK is aight. (But — obviously — fair point 😉)